sklearn/feature_selection/from_model.py

```diff
@@ -199,9 +215,19 @@ def _get_support_mask(self):
                 'Either fit the model before transform or set "prefit=True"'
                 ' while passing the fitted estimator to the constructor.')
         scores = _get_feature_importances(estimator)
-        self.threshold_ = _calculate_threshold(estimator, scores,
-                                               self.threshold)
-        return scores >= self.threshold_
+        mask = np.zeros(scores.shape, dtype=bool)
```

Comment:
Didn't we decide that we should return the intersection of the `threshold` and the `max_n_features`?